src/store.ts

```diff
@@ -328,6 +328,20 @@ export class Store<T> {
       this.devTools.subscribe((message: any) => {
         this.logger[getLogType(this.options, "devToolsStatus", LogLevel.debug)](`DevTools sent change ${message.type}`);
 
+        if (message.type === "ACTION" && message.payload) {
+          const byName = Array.from(this.actions).find(function (_a) {
+            return _a[0].name === message.payload.name;
+          });
+          const action = this.lookupAction(message.payload.name) || byName && byName[0];
+
+          if (!action) {
+            throw new Error("Tried to remotely dispatch an unregistered action");
+          }
+
+          this.dispatch(action, ...message.payload.args.slice(1));

```

Comment:
for whatever reason message.payload.args contains only strings, even if numbers or complex objects are sent via the devtools

Reply:
@jmzagorski, @dannyBies and @Vheissu perhaps one of you guys has an idea what could be wrong?

Reply:
I haven't been able to test it but maybe it's related to https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#serialize

Reply:
I thought so as well and tried all combinations, but to no avail

Reply:
ok I think I got what they're up to. One needs to JSON.parse every provided argument individually. Fixed it with the new update